experimental/dds/tree2/src/feature-libraries/sequence-field/format.ts

```diff
@@ -164,29 +61,14 @@ export const HasRevisionTag = Type.Object({ revision: Type.Optional(RevisionTagS
  * Carries a `MoveId` in case it is rebased over the content being moved out, in which case this mark
  * will transform into a pair of returns which will move the content back into this cell.
  */
-export interface Insert extends HasMoveId, HasRevisionTag {
-	type: "Insert";
-}
-export const Insert = Type.Composite(
-	[
-		HasMoveId,
-		HasRevisionTag,
-		Type.Object({
-			type: Type.Literal("Insert"),
-		}),
-	],
-	noAdditionalProps,
-);
+const Insert = Type.Composite([HasMoveId, HasRevisionTag], noAdditionalProps);
```

Comment:
In general, not sure how I feel about duplicating comments in this file with the ones in `types`. We do expect the semantics of fields here to diverge with those in `types.ts` over time, but not sure what exactly is the most helpful to document.

I've used *some* discretion on which comments to preserve, but wasn't particularly careful about it.